code/modules/admin/verbs/map_template_loadverb.dm

```diff
@@ -15,7 +15,9 @@
 
 	var/list/preview = list()
 	for(var/S in template.get_affected_turfs(T,centered = TRUE))
-		preview += image('icons/turf/overlays.dmi',S,"greenOverlay")
+		var/image/item = image('icons/turf/overlays.dmi',S,"greenOverlay")
+		item.plane = ABOVE_LIGHTING_PLANE
```

Comment:
Modifying images is significantly more costly than mutable appearances, since you need to do that here now you should switch to that and probably also save the image so it doesn't have to be recreated.

Alternatively use the layer arg in image()

Reply:
It has to set a plane to get it to differ from the plane of the space turf it's placed on, but it should be easy to create a static `mutable_appearance` instead.

Reply:
it think it was 3 modifications before the overhead of creating the mutable_appearance is worth it. at that point you might as well make a static image since you don't need to do 3 changes.

Reply:
On examination, I'm not sure the image can be made static given that its `loc` is actually being used and the image is one of many being added to `client.images`.

Reply:
ya, using mutable_appearances is really a overlay thing, client images have to be images.